pose_format/pose.py

```diff
@@ -63,13 +65,10 @@ def normalize(self, info: PoseNormalizationInfo, scale_factor: float = 1):
 
         mean_distance = distance_batch(p1s, p2s).mean()
 
-        if mean_distance == 0.0:
-            scale = scale_factor
-        else:
-            scale = scale_factor / mean_distance  # scale all points to dist/scale
```

Comment:
is this because this branch can't be predicted for graph mode?

Reply:
this works even in graph mode, but not if a function containing this section is called by `tf.data.Dataset.map`. I am open to other, better solutions